src/utils/customPropTypes.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash';
+
+let customPropTypes = {
+  mutuallyExclusive: (exclusives, props, propName, componentName) => {
+    _.forEach(exclusives, exclusiveProp => {
+      if (props[propName] && props[exclusiveProp]) {
+        throw new Error(`\`${componentName}\` should only have one of type \`${propName}\` or \`${exclusiveProp}\`, ` +
```

Comment:
Had to `throw` error as opposed to `return` because the new Error needed to attach itself to the prop, not the utility property `mutuallyExclusive`